packages/types/src/CType.ts

```diff
@@ -34,7 +35,7 @@ export interface ICTypeSchema {
 export type CTypeSchemaWithoutId = Omit<ICTypeSchema, '$id'>
 
 export interface ICType {
-  hash: string
+  hash: HexString
```

Comment:
Should either this reference `CtypeMetadata` or the other way round (most likely)?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/505/commits/6f20a1067af5155eb85d510a308bdf96c8d58e21